modules/installation-special-config-storage.adoc

```diff
@@ -0,0 +1,156 @@
+
+.. Set up or access an existing Tang server. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption_security-hardening#network-bound-disk-encryption_configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption[Network-bound disk encryption] for instructions. See link:https://youtu.be/2uLKvB8Z5D0[Securing Automated Decryption New Cryptography and Techniques] for a presentation on Tang.
+
+.. Install the `clevis` package, if it is not already installed:
++
+[source,terminal]
+----
+$ sudo yum install clevis -y
+----
+
+.. Generate a thumbprint for the Tang server.
+
+... In the following command, replace the value of `url` with the Tang server URL:
++
+[source,terminal]
+----
+$ echo nifty random wordwords | clevis-encrypt-tang '{"url":"https://tang.example.org"}'
+----
++
+.Example output
+[source,terminal]
+----
+The advertisement contains the following signing keys:
+
+PLjNyRdGw03zlRoGjQYMahSZGu9
+----
+
+... When the `Do you wish to trust these keys? [ynYN]` prompt displays, type `Y`, and the thumbprint is displayed:
++
+.Example output
+[source,terminal]
+----
+eyJhbmc3SlRyMXpPenc3ajhEQ01tZVJiTi1oM...
+----
+
+.. If nodes are configured with static IP addressing, ensure you use the coreos-installer `--append-karg` option when installing {op-system} nodes to set the IP address of the installed system.  Append the `ip=` and other arguments needed for your network.
++
+[IMPORTANT]
+====
+Some methods for configuring static IPs do not affect the initramfs after the first boot, and so will not work with Tang encryption.  These include the coreos-installer `--copy-network` option, and adding `ip=` arguments to the kernel command line of the live ISO or PXE image during installation.  Incorrect static IP configuration causes the second boot of the node to fail.
+====
+
+. Change to the directory that contains the installation program and generate the Kubernetes manifests for the cluster:
++
+[source,terminal]
+----
+$ ./openshift-install create manifests --dir=<installation_directory>
+----
+
+. Create a Butane config configuring disk encryption and/or mirroring. For example, to configure storage for worker nodes, create `$HOME/clusterconfig/worker-storage.bu`:
++
+[source,yaml]
+.Butane config example
+----
+variant: openshift
+version: 4.8.0
+metadata:
+  name: worker-storage <1>
+  labels:
+    machineconfiguration.openshift.io/role: worker <1>
+boot_device:
+  layout: x86_64 <2>
+  luks: <3>
+    tpm2: true <4>
+    tang: <5>
+      - url: https://tang.example.com
+        thumbprint: eyJhbmc3SlRyMXpPenc3ajhEQ01tZVJiTi1oM...
+    threshold: 1 <6>
+  mirror: <7>
+    devices: <8>
+      - /dev/sda
+      - /dev/sdb
+openshift:
+  fips: true <9>
+----
++
+<1> On master nodes, substitute `master` for `worker` in both of these locations.
```

Comment:
Similar comments:
"On control plane nodes, substitute `master` (or control plane) for `worker` in both of these locations.